deeplearning4j-scaleout/dl4j-streaming/src/main/spark-2/org/deeplearning4j/streaming/pipeline/spark/StreamingContextUtils.java

```diff
@@ -0,0 +1,27 @@
+package org.deeplearning4j.streaming.pipeline.spark;
+
+import org.apache.spark.api.java.JavaRDD;
+import org.apache.spark.api.java.function.VoidFunction;
+import org.apache.spark.streaming.api.java.JavaDStream;
+import org.apache.spark.streaming.api.java.JavaStreamingContext;
+
+/**
+ * In order to handle changes between Spark 1.x and 2.x
+ */
+public class StreamingContextUtils {
+
+    public static void awaitTermination(JavaStreamingContext jssc, long timeout) {
+        try {
+            if(timeout < 0)
+                jssc.awaitTermination();
+            else
+                jssc.awaitTerminationOrTimeout(timeout);
+        } catch (InterruptedException e) {
```

Comment:
Yes, the argument of `awaitTermination` disappears in 2.1.x. But since you only use `awaitTermination` in the case where you have no applicable `timeout` being passed, that's not really an issue, is it ? You could:
```
try {
  (timeout < 0) ? jssc.awaitTermination() : jssc.awaitTerminationOrTimeOut(timeout);
} catch (Exception e) {
  if (e instanceof InterruptedException) {
    // handle exception
  }
  ```

no?